### ISSUE TITLE
Metadata service: Spatial data Service Category (MD SDS 3.4)

### ISSUE THREAD

**Issue by idevisser** [INSPIRE-MIF/helpdesk-validator#84]
I noticed a problem with the Spatial data Service Category. 

In the TG the following requirement is  written;

TG Requirement 3.4: metadata/2.0/req/sds/sds-category
At least one Spatial Data Service category or subcategory for the described service shall be given using the language-neutral keyword values as defined in Part D 4 “Classification of Spatial Data Services” of [Regulation 1205/2008].

We can't use the INSPIRE code list defined on

http://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceCategory

because the values are here translated to the used language, in our case Dutch. The Dutch values don't fulfil the  TG requirement to provide the language-neutral keyword value

This problem can be solved on two ways;

remove the requirement to provide the language neutral keyword value
or 
replace  the translations of the values in the codelist with the laguag nutral values. (Defenitions can still be in one of the MS languages).

**Comment by idevisser** [comment]
XML Example which is not accepted :

<gmd:keyword>
		<gmx:Anchor xlink:href="http://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceCategory/comTransferService">Dienst gegevensoverdracht</gmx:Anchor>
</gmd:keyword>

**Comment by skafka** [comment]
We met the same problem. Now, when the language neutral URIs are used in anchors, I think the right way is to validate anchor URI at first and only if not available, the string itself. It should be at all places where the Anchor is used.

**Comment by josemasensio** [comment]
Dear @idevisser,

Thank you for your message.

Could you provide us the metadata you are testing please? Thank you in advance.

Best regards.

**Comment by idevisser** [comment]
[Test_Metadata_Services_2019_serviceCategory.zip](https://github.com/inspire-eu-validation/community/files/3385690/Test_Metadata_Services_2019_serviceCategory.zip)

herby a complete XML

**Comment by pvgenuchten** [comment]
see also https://inspire.ec.europa.eu/forum/discussion/view/261142/what-is-meant-with-language-neutral-value-tg-metadata-20

**Comment by inakidiazdecerio** [comment]
Dear @idevisser,

The keyword value is one of the language-natural keyword values as defined in Classification of Spatial Data Services Part D 4 (https://eur-lex.europa.eu/LexUriServ/LexUriServ.do?uri=OJ:L:2008:326:0012:0030:EN:PDF). For example, inyour case it should be:

<gmd:keyword>
<gmx:Anchor xlink:href="http://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceCategory/comTransferService">comTransferService</gmx:Anchor>
</gmd:keyword>

Let us know if you still have any question.

Best Regards

**Comment by skafka** [comment]
The idea was to use e.g.

```
<gmd:keyword
<gmx:Anchor xlink:href="http://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceCategory/comTransferService">Transfer service</gmx:Anchor>
</gmd:keyword>
```

and validate only URI rather than string itself!

**Comment by josemasensio** [comment]
Dear @idevisser,
Does the solution worked?

Regards.

**Comment by fantamir** [comment]
Pleas provide the information when the solution will be released and implemented in metadata validator.  Thank you very much.

**Comment by skafka** [comment]
Still looks like the text is validated instead of URI on both the official validator and staging instance. 
The test record is here: https://dev.bnhelp.cz/projects/micka6/record/xml/5a4c757b-30c8-4edf-ab2f-40f5585671f1 
See the validation explanation:

![image](https://user-images.githubusercontent.com/11944561/69817244-79222b00-11fa-11ea-9e39-ef8a7113f92d.png)

**Comment by josemasensio** [comment]
Dear @skafka,

Thank you for your message. Regarding this issue, it was deployed and closed in v1.0.8, so it is no longer tracked. Please, could you open a new issue to be able to do a follow-up of and this way when solving it to be able to put it in the reference of the changelog?

Thank you and best regards.
